tests/unit/type/hash-field-expire.tcl

```diff
@@ -1213,8 +1213,8 @@ start_server {tags {"external:skip needs:debug"}} {
         test "HSETEX - Test 'PXAT' flag ($type)" {
             r del myhash
             r hset myhash f1 v2 f2 v2 f3 v3
-            assert_equal [r hsetex myhash PXAT 4000000000000 FIELDS 1 f2 v2] 1
-            assert_range [expr [r httl myhash FIELDS 1 f2] + [clock seconds]] 3900000000 4000000000
+            assert_equal [r hsetex myhash PXAT [expr [clock milliseconds] + 10000] FIELDS 1 f2 v2] 1
+            assert_range [r httl myhash FIELDS 1 f2] 5 10
```

Comment:
maybe we can fix other instances as part of this PR, if you search `+ [clock seconds]` in the file, there are a few more.

Reply:
done [ca4d2d6](https://github.com/redis/redis/pull/13829/commits/ca4d2d6a3cda81388236629966dda222639915ba)
i think i should do that, but i don't know why i forgot.


there's one left, i think it doesn't matter
```
assert_error {ERR invalid expire time in 'hexpireat' command} {r HEXPIREAT myhash [expr (1<<48) / 1000 + [clock seconds] + 100] FIELDS 1 f1}
```